sensors/01-meteorological-data.Rmd

```diff
@@ -68,31 +59,31 @@ The key is to process each type of met data (site, reanalysis, forecast, climate
 
 ### The Geostreams Database
 
-!(schema)[https://cloud.githubusercontent.com/assets/9286213/16991300/b2f2b09a-4e60-11e6-96b7-8b63c3d1f995.jpg]
+![Schema](https://cloud.githubusercontent.com/assets/9286213/16991300/b2f2b09a-4e60-11e6-96b7-8b63c3d1f995.jpg)
 
 #### Querying the API
 
-key query terms:
-
-* /sensors
-  * sensor_name=''
-* /streams
-  * 
-* /datapoints
-  * &since=YYYY-MM-DD
-  * &until=YYYY-MM-DD
-  * stream_id=
+The data can be accessed using a URL. You can find data for a particular station, 
+a certain sensor from a station, or the datapoints from a sensor. Below are
+the URLs for the UA-MAC AZMET weather station, a certain sensor at that station, 
+and datapoints from that sensor for the month of January 2017. 
```

Comment:
Maybe change this to be May 2018 to keep in line with the data we'll have available?

Reply:
See c28bf31c8e165803838d90d909219e56e83f5e7a, though it looks like there's only weather data available for the month of April?